### ISSUE TITLE
Mr. Transcript

### ISSUE THREAD

**Issue by colelin26** [wattools/WatTools#56]
What category should your request go under?

```
    {
      "Name": {
        "text": "Mr. Transcript",
        "url": "https://wa-transcript.herokuapp.com/"
      },
      "Description": {
        "text": "
Web application for UW students to scrape transcript data and calculate 4.0 GPA."
      },
      "Source Code": {
        "text": "Github Link",
        "url": "https://github.com/colelin26/wat-transcript"
      },
      "Authors": {
        "text": "Cole Lin"
      }
    }
```

**Comment by GeraldNDA** [comment]
Ripp I made one too: https://geraldnda.github.io/uwaterloo-gpa-calculator/

**Comment by BrewedCoffee** [comment]
Looks like the WATranscript site has moved to https://wa-transcript.herokuapp.com/.

What category would you say GPA converters fall under? We might need to take a second look at our classifications in general...

**Comment by colelin26** [comment]
@BrewedCoffee Thanks for looking into this. 

Yes, the website was moved to https://wa-transcript.herokuapp.com/.  I also somehow decided to rename it to `Mr. Transcript` at the end.

It might fall under "General" in this case based on what other things are under general? - like tools that help studying at Waterloo in general? Or we can rename `Course Selection` to `Courses` and make it under `Courses`?

My code in this project isn't very clean because I was using this project to learn React at the time, but I tested with my latest transcript and I think the functionality should still be good.
